package.json

```diff
@@ -549,13 +549,14 @@
   },
   "dependencies": {
     "@lit-labs/observers": "^2.0.2",
-    "@malloydata/db-bigquery": "0.0.116",
-    "@malloydata/db-duckdb": "0.0.116",
-    "@malloydata/db-postgres": "0.0.116",
-    "@malloydata/malloy": "0.0.116",
-    "@malloydata/malloy-sql": "0.0.116",
-    "@malloydata/render": "0.0.116",
-    "@malloydata/syntax-highlight": "0.0.116",
+    "@malloydata/db-bigquery": "0.0.118",
+    "@malloydata/db-duckdb": "0.0.118",
+    "@malloydata/db-postgres": "0.0.118",
+    "@malloydata/malloy": "0.0.118",
+    "@malloydata/malloy-sql": "0.0.118",
+    "@malloydata/render": "0.0.118",
+    "@malloydata/syntax-highlight": "0.0.118",
+    "@vscode/codicons": "^0.0.33",
```

Comment:
This was deleted in main

Reply:
Oops, I screwed up the rebase. I will drop them now